mat64/format_example_test.go

```diff
@@ -37,10 +38,10 @@ func ExampleFormatted() {
 	//     ⎢.  4  5⎥
 	//     ⎣.  .  6⎦
 	//
-	// after modification with only non-zero values:
-	// a = ⎡1  2  .⎤
-	//     ⎢.  4  5⎥
-	//     ⎣.  .  6⎦
+	// after modification for scientific notation:
```

Comment:
This breaks the semantics of this sentence. Please do this separately rather than just replacing the zeroing example.